src/test/kotlin/io/bazel/worker/WorkerEnvironment.kt

```diff
@@ -87,6 +92,13 @@ object WorkerEnvironment {
       }
     }
 
+    fun waitForStdOut() {
+      while (stdOut.input.available() != 0) {
+        Thread.onSpinWait()
+        Thread.sleep(1) // don't starve other processes.
```

Comment:
Doesn't doing a `Thread.sleep(1)` more or less do as much as `onSpinWait()` or more? As in, if you have `sleep(1)` do you still need `onSpinWait()`?